src/main/java/ua/edu/internship/interview/data/repository/UserQuestionRepository.java

```diff
@@ -0,0 +1,18 @@
+package ua.edu.internship.interview.data.repository;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import ua.edu.internship.interview.data.documents.UserQuestionDocument;
+
+import java.util.List;
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'java.util.List'
<!-- __reviewdog__:Cg85NTY0YWQ2NjI5MDk1ZjgSCmNoZWNrc3R5bGU= -->